src/pipecat/processors/aggregators/llm_response.py

```diff
@@ -109,6 +110,7 @@ async def _push_aggregation(self):
 
             # Reset
             self._aggregation = ""
+            self._aggregating = False
             self._seen_start_frame = False
             self._seen_end_frame = False
             self._seen_interim_results = False
```

Comment:
We have a bug where we append the previous user speech a second time, because the "Reset" block here comes after the `await self.push_frame()`

Example messages from the buggy case:
```
  {
    "role": "user",
    "content": " Tell me a long story about a horse."
  },
  {
    "role": "assistant",
    "content": " Certainly!"
  },
  {
    "role": "user",
    "content": " Tell me a long story about a horse. Oh, wait. I changed my mind. I want you to"
  },
```